controlsfx/src/main/java/impl/org/controlsfx/skin/SearchableComboBoxSkin.java

```diff
@@ -0,0 +1,341 @@
+/**
+ * Copyright (c) 2017, 2019 ControlsFX
```

Comment:
The copyright should show the year when the file was added and the year when it was last updated.

For this file it will be just:

> Copyright (c) 2019, ControlsFX

Reply:
updated